Readme files for codes without one (#2)
shuttle-hq/shuttle-examples: axum/websocket/README.md

> # Web Sockets with AXUM 
> This example shows how to use axum to develop a web socket [axum](https://docs.rs/axum/0.1.1/axum/ws/index.html).
> The idea is that the `lib.rs` file handles the client and user and http requests and opens the web sockets.

Reviewer: ```suggestion
# Web Sockets with axum 
This example shows how to use axum to develop a web socket [axum](https://docs.rs/axum/0.1.1/axum/ws/index.html) application which we can deploy with shuttle..
```
